client/src/document/baseline-indicator.tsx

```diff
@@ -7,10 +7,11 @@ import { useLocation } from "react-router";
 
 import "./baseline-indicator.scss";
 
-import type {
-  SupportStatus,
-  browserIdentifier,
-} from "../../../libs/types/web-features";
+// web-features doesn't export these types directly so we need to do a little typescript magic:
+import type { features } from "web-features";
+type SupportStatus = (typeof features)[keyof typeof features]["status"];
+type browserIdentifier =
```

Comment:
Nit: Types should start with uppercase letter, I guess.

```suggestion
type BrowserIdentifier =
```

Reply:
the type is `browserIdentifier` upstream, so I don't want to make this change, as it'll require a change back when we import them directly

Reply:
Checked the [Google TypeScript Style Guide](https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type) and - not suprisingly - it suggests `UpperCamelCase` for types. It should be `BrowserIdentifier` upstream.

Reply:
I'll make that change in my PR upstream, so I'll update here as well